.github/workflows/ci.yml

```diff
@@ -103,6 +103,7 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+          always_pull: true
```

Comment:
If we're afraid of Docker's new pull limits, we could only pull if branch = master.
Any opinions?

Reply:
I don't think this option has any effect. AFAIK GitHub doesn't cache images.

Reply:
If your assumption is true, it shouldn't harm to add it.